### DISCUSSION TITLE
关于 mineadmin amqp 相关功能的废除标记解释

### DISCUSSION THREAD

**Discussion by zds-s** [mineadmin/MineAdmin#162] [Announcements]
当时是适配 hyperf 2.2 - 3.0 的延迟推送 外加现在基本没人用，在 2.0 正式版会删除这块类库
 后期会改造一个基于 redis 的消息推送通知

**Comment by taobali32** [comment]
. 留着不就好了， 在复制一份出来搞个redis消息推送通知的
